lib/esm/generated/telemetry.d.ts

```diff
@@ -127,6 +127,14 @@ export declare type TelemetryConfigurationEvent = CommonTelemetryProperties & {
              * Whether sessions across subdomains for the same site are tracked
              */
             track_session_across_subdomains?: boolean;
+            /**
+             * Whether resources are tracked
+             */
+            track_resources?: boolean;
+            /**
+             * Whether long tasks are tracked
+             */
+            track_long_task?: boolean;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You may want to add these new attributes to the configuration samples used for validation